dev-python/Rx/Rx-3.1.0.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_6 python3_7 python3_8 python3_9 )
+
+inherit distutils-r1 virtualx
+
+MY_P="RxPY"
+DESCRIPTION="Reactive Extensions for Python"
+HOMEPAGE="http://reactivex.io/"
+SRC_URI="https://github.com/ReactiveX/${MY_P}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/${MY_P}-${PV}"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+IUSE="test"
+DEPEND="
+	test? (
+		dev-python/coverage[${PYTHON_USEDEP}]
```

Comment:
You aren't using it, and you aren't supposed to.

Reply:
Sorry, I just put it in because `setup.py` told me to (in `tests_require`). Removed in e9ef2183cb0e38725b6e0bcd6c30f22cdc3cfb43.